designer_v2/lib/features/study/study_test_controller.dart

```diff
@@ -16,14 +16,17 @@ part 'study_test_controller.g.dart';
 class StudyTestController extends _$StudyTestController {
   @override
   StudyTestControllerState build(StudyID studyId) {
+    // final studyRepo = ref.read(studyRepositoryProvider);
     ref.onDispose(() {
       // Reload the study after disposing the test controller so that any
       // data changes resulting from testing are reflected in the study for
       // other parts of the app (e.g. test data that was generated)
       //
       // Ideally, we would stream changes from the backend/database directly,
-      // but this is a sufficient workaround for now
-      ref.watch(studyRepositoryProvider).fetch(studyId);
+      // but this is a sufficient workaround for now.
+
+      // TODO test if this is needed or the study gets already updated after closing the test view
+      // studyRepo.fetch(studyId);
```

Comment:
Using network inspection of chrome the study gets updated twice. Check where the update comes from and if this update here is needed. Do this by creating example test data inside the app using the test screen (both intervention test split screen and test page) and then see if the data can be downloaded in the results page